modules/dastool/scaffolds2bin/main.nf

```diff
@@ -0,0 +1,46 @@
+include { initOptions; saveFiles; getSoftwareName; getProcessName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process DASTOOL_SCAFFOLDS2BIN {
+    tag "$meta.id"
+    label 'process_low'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    conda (params.enable_conda ? "bioconda::das_tool=1.1.3" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/das_tool:1.1.3--r41hdfd78af_0"
+    } else {
+        container "quay.io/biocontainers/das_tool:1.1.3--r41hdfd78af_0"
+    }
+
+    input:
+    tuple val(meta), path(fasta)
+    val(extension)
+
+    output:
+    tuple val(meta), path("*.scaffolds2bin.tsv"), emit: scaffolds2bin
+    path "versions.yml"                         , emit: versions
+
+    script:
+    def prefix = options.suffix ? "${meta.id}${options.suffix}" : "${meta.id}"
+    def file_extension = extension ? extension : "fasta"
```

Comment:
```suggestion
    def file_extension = extension ? extension : "fasta"
```

Is this necessary? Default is `fasta` anyway, so in principle `-e` should be passed via `option.args`

Reply:
I would keep, so it's in the `meta.yml`. I can already see my future self looking for the reason why Das Tool is not accepting my `.fasta` files.

Reply:
OK fair enough. No harm I guess...